internal/broker/config.go

```diff
@@ -24,15 +27,55 @@ const (
 
 	// usersSection is the section name in the config file for the users and broker specific configuration.
 	usersSection = "users"
+	// allowedUsersKey is the key in the config file for the users that are allowed to access the machine.
+	allowedUsersKey = "allowed_users"
+	// ownerKey is the key in the config file for the owner of the machine.
+	ownerKey = "owner"
 	// homeDirKey is the key in the config file for the home directory prefix.
 	homeDirKey = "home_base_dir"
 	// SSHSuffixKey is the key in the config file for the SSH allowed suffixes.
 	sshSuffixesKey = "ssh_allowed_suffixes"
+
+	// AllUsersKeyword is the keyword for the `allowed_users` key that allows access to all users.
+	AllUsersKeyword = "ALL"
+	// OwnerUserKeyword is the keyword for the `allowed_users` key that allows access to the owner.
+	OwnerUserKeyword = "OWNER"
+
+	// ownerAutoRegistrationConfigPath is the name of the file that will be auto-generated to register the owner.
+	ownerAutoRegistrationConfigPath     = "20-owner-autoregistration.conf"
+	ownerAutoRegistrationConfigTemplate = "templates/20-owner-autoregistration.conf.tmpl"
+)
+
+var (
+	//go:embed templates/20-owner-autoregistration.conf.tmpl
+	ownerRegistrationConfig embed.FS
```

Comment:
nitpick: missed this one (which I didn't point to before) for renaming to `ownerAutoRegistretionConfig`, feel free to disregard

Reply:
oops, changed it